specs/phase0/fork-choice.md

```diff
@@ -68,10 +73,34 @@ Any of the above handlers that trigger an unhandled exception (e.g. a failed ass
 ```python
 @dataclass(eq=True, frozen=True)
 class LatestMessage(object):
-    epoch: Epoch
+    slot: Slot
     root: Root
 ```
 
+#### `BlockSlotNode`
+
+```python
+class BlockSlotNode(Container):
+    block_root: Root
+    slot: Slot
+    parent_node: Root
+```
+
+#### `BlockSlotKey`
+
+```python
+class BlockSlotKey(Container):
+    block_root: Root
+    slot: Slot
+```
+
+#### `get_block_slot_key`
+
+```python
+def get_block_slot_key(block_root: Root, slot: Slot) -> Root:
```

Comment:
May just be me, but I'd expect a function with this name to return the BlockSlotKey class rather than its root.